src/solc_0.8/polygon/child/land/PolygonLand.sol

```diff
@@ -9,8 +9,16 @@ import "./PolygonLandBaseToken.sol";
 contract PolygonLand is PolygonLandBaseToken {
```

Comment:
Let's do some kind of versionning like Asset
PolygonAssetV2
for instance
PolygonLandV1